api/src/main/resources/workbench-api.yaml

```diff
@@ -6447,6 +6461,22 @@ definitions:
         type: string
         description: 'The user''s detailed Institutional Role at this Institution,
           as text, if its enumerated type is OTHER'
+  InstitutionTierRequirement:
+    type: object
+    description: Institution-specific access tier membership requirement.
+    required:
+      - accessTierShortName
+      - requirementEnum
+    properties:
+      accessTierShortName:
+        type: string
+        description: The short unique name for the access tier.
+      requirementEnum:
```

Comment:
nit: enum suffix feels a bit redundant here to me, it is already typed as such

Also, should this be named something more specific, like `emailRequirement` ? throughout?

Reply:
requirement seems to generic? How about requirementName? Or something else. emailRequirement seems not including `NO_ACCESS` case. (e.g. user switch `enable CT access` toggle to false.)

Reply:
What is the unifying concept here? At a glance I'm not sure since DOMAINS and NO_ACCESS don't seem very similar to me. `membershipRequirement` ? `accessRequirement` ?

Reply:
The enum has 3 value: EMAILS, DOMAINS, NO_ACCESS. Emails similar to DuaType.INDIVIDUAL.  DOMAINS is simialr to today's  DuaType.MASTER. NO_ACCESS means no access at all. (inst admin toggle "disable CT access", that API will set the enum to NO_ACCESS). 
Both name LG, will pick one from them! Thanks!